### PR TITLE
Update gem dependencies and ReadMe

### PR DESCRIPTION
**Pull request by laurenherdman**
Updates gem dependencies

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,5 +6,5 @@ rvm:
   - '2.2.4'
   - '2.3.0'
 script:
-  - bundle exec rubocopter --commit origin/master
+  - gem install rubocop
```
Comment (michaelfich) [inline .travis.yml]: this appears to swap out *running* rubocopter for *installing* rubocop.  is this intended?

Comment (mbasset) [inline .travis.yml]: I think there is a specific step for installing things. Once installed you should run `rubocop` in this step.

**Comment by mbasset** [comment]
@laurenherdman Please also add ruby 2.4.1 to the list of tested rubies.

File: .travis.yml [hunk 2]
```diff
@@ -5,6 +5,11 @@ rvm:
   - '2.1.8'
   - '2.2.4'
   - '2.3.0'
+  - '2.4.1'
+
+before_install:
```
Comment (michaelfich) [inline .travis.yml]: this is nitpicking a little but, but may want to change this from `before_install` to `install`.  i think installing stuff like `rubocop` is what the `install` event hook is for.  heres the docs for the [travis config file](https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Installation-Step)

File: .idea/.rakeTasks [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mbasset) [inline .idea/.rakeTasks]: I think you need to globally ignore the .idea folder with git.

**Comment by mbasset** [comment]
@laurenherdman looks good except that when using activesupport > 5 it does not work on older rubies.

**Pull request merged**